src/checker/mod.rs

```diff
@@ -108,6 +108,18 @@ where
         }
     }
 
+        if config.is_enabled(Detector::Reflow) {
```

Comment:
while we use the the checkable trait, we should not mix this with other detectors, which are going to run concurrently - use a separate flag in the `Args` struct.